### DISCUSSION TITLE
IBluetoothLE support on WinUI

### DISCUSSION THREAD

**Discussion by axa88** [dotnet-bluetooth-le/dotnet-bluetooth-le#819] [Q&A]
Looking through the source code I believe Ive found evidence that `IBluetoothLE` is implemented on `WinUI`.
Yet I dont seem to have any functionality on my Windows device.
None of the 4 properties nor `StateChanged` event seem to work.

Using the latest stable v3.0.0
Am i missing something?

**Comment by AskBojesen** [comment]
Yes - Go with the latest NuGet from master. Check preview checkbox in Visual Studio or download from GHA: 

https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/actions 

I refactored a lot and fixed a lot in the Windows implementation  - the latest is the best.
Download the latest build from artifacts on master:
https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/actions/runs/7947538628/artifacts/1254330309

**Reply by axa88** [reply]
I'll ~~likely~~ pull this ~~today~~ in the next couple is days
